CNAME

```diff
@@ -0,0 +1 @@
+charts.jaegertracing.io
```

Comment:
We'd need the Jaeger team to create this. You probably have already but for ref see https://help.github.com/en/github/working-with-github-pages/configuring-a-custom-domain-for-your-github-pages-site